pm_common/portmidi.h

```diff
@@ -904,7 +904,7 @@ PMEXPORT PmError Pm_Poll(PortMidiStream *stream);
 
     @param length the length of the \p buffer.
 
-    @return TRUE, FALSE, or an error value.
+    @return #pmNoError or an error code.
```

Comment:
Thanks -- you are correct. I added a list of possible error codes and what they mean to this, Pm_Write and Pm_WriteSysEx, and I'll commit everything after some more pull request reviewing and testing.